### ISSUE TITLE
Interval for collecting FPS metric in gst-shark

### ISSUE THREAD

**Issue by alekseyfa** [RidgeRun/gst-shark#75]
Hi,

I use framerate tracers of gst-shark. I would like to know if it is possible to set the interval for collecting FPS metrics? I want traces to be reported every nth second.  How can I use bigger interval for data collection?

**Comment by michaelgruner** [comment]
This is a feature I've always wanted. As per now is not possible, let me see if I can work out quickly. I'll be back in a few hours.

**Comment by michaelgruner** [comment]
Hey @alekseyfa I've implemented this in #78 (the featuer/configurable-period branch). Would you mind giving it a try and reporting your results? Basically you pass in the period as the following:

```bash
GST_DEBUG=GST_TRACER:7 GST_TRACERS="framerate(period=5);cpuusage(period=3)" gst-launch-1.0 ...
```

The change applies to all periodic tracers: framerate and cpuusage as per now.
